test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -0,0 +1,23 @@
+# This is a sample .feature file
+# Squish feature files use the Gherkin language for describing features, a short example
+# is given below. You can find a more extensive introduction to the Gherkin format at
+# https://cucumber.io/docs/gherkin/reference/
+Feature: Status Desktop Chat
+
+    As a user I want to join a room and chat.
+
+    The following scenarios cover basic chat flows.
+
+    Background:
+         Given A first time user lands on the status desktop and generates new key
+    	 When user inputs username tester123 and password TesTEr16843/!@00
+    	 Then the user lands on the signed in app
+
+    Scenario: User joins a room and chats
+		 When user joins chat room test
+		 Then user is able to send chat message
```

Comment:
For me maybe this then should be the `When` as it involves the action we want to test and the `Then` should be checking if the messages have been sent, so picking up the messages from the chat view and comparing sent with received / published in the chat conversation.